src/test/java/uk/gov/digital/ho/hocs/audit/service/DocumentExportServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.digital.ho.hocs.audit.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.digital.ho.hocs.audit.client.info.dto.CaseTypeDto;
+import uk.gov.digital.ho.hocs.audit.core.utils.ZonedDateTimeConverter;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDate;
+import java.util.Set;
+
+import static org.mockito.BDDMockito.given;
+
+public class DocumentExportServiceTest extends BaseExportServiceTest {
+
+    @Autowired
+    private DocumentExportService documentExportService;
+
+    private ZonedDateTimeConverter zonedDateTimeConverter;
+
+    @BeforeEach
+    public void setup() {
+        zonedDateTimeConverter = new ZonedDateTimeConverter();
+
+        given(infoClient.getCaseTypes()).willReturn(Set.of(new CaseTypeDto("Test", "a1", "TEST")));
+    }
+
+    @Test
+    public void shouldReturnExport() throws IOException {
+        documentExportService.export(LocalDate.of(2020, 1, 1), LocalDate.now().plusDays(1), outputStream, "TEST",
```

Comment:
There are no `DOCUMENT_CREATED` events in [setup.sql](https://github.com/UKHomeOffice/hocs-audit/blob/b4b3b15d83fdfb518a90580a7e67c6b08580a32f/src/test/resources/export/setup.sql) so this isn't fully testing that events can be mapped to an export row. Note: it looks like `getCsvDataRows(result)` doesn't exclude the header row, hence that returning size = 1.

Reply:
I'd still like to see some actual data being mapped in the test. Consider adding something like 

```
(5, '00000000-0000-0000-0000-000000000005', '10000000-0000-0000-0000-000000000000',
 '20000000-0000-0000-0000-000000000000', '30000000-0000-0000-0000-000000000004', 'service-1',
 '{"documentUUID": "30000000-0000-0000-0000-000000000000", "documentType": "documentType", "documentTitle": "Document title"}',
 'env-1', '2020-01-01 01:00:00.000000', 'DOCUMENT_CREATED', '40000000-0000-0000-0000-000000000000', 'a1');
```

to [setup.sql](https://github.com/UKHomeOffice/hocs-audit/blob/b4b3b15d83fdfb518a90580a7e67c6b08580a32f/src/test/resources/export/setup.sql) and check that the 2nd data row includes the expected values?

Reply:
I think I may be missing something, is that not what I'm already checking here on line 47?

Reply:
Apologies that change wasn't showing for me when I reviewed it. LGTM 👍